libeos-payg/manager.c

```diff
@@ -88,6 +88,12 @@ static void        shutdown_save_state_cb (GObject      *source_object,
 static guint64     epg_manager_get_expiry_time     (EpgProvider *provider);
 static gboolean    epg_manager_get_enabled         (EpgProvider *provider);
 static guint64     epg_manager_get_rate_limit_end_time (EpgProvider *provider);
+static EpgClock *  epg_manager_get_clock (EpgProvider *provider);
+
+static guint64 last_save_time; /* wallclock timestamp of last state save */
+static guint64 last_save_expiry; /* seconds left to expiration at time of last state save */
+static gboolean last_save_time_set = FALSE; /* whether last_save_time has been set */
+static gboolean last_save_expiry_set = FALSE; /* whether last_save_expiry has been set */
```

Comment:
Move these static variables inside the `EpgManager` object, or there’s no point in having an object.